src/hooks/__internal__/useInputAccessibility/useInputAccessibility.ts

```diff
@@ -1,40 +1,47 @@
 export default function useInputAccessibility({
   id,
+  validationRedesignOptIn,
   error,
   warning,
   info,
   label,
   fieldHelp,
 }: {
   id: string;
+  validationRedesignOptIn?: boolean;
   error?: string | boolean;
   warning?: string | boolean;
   info?: string | boolean;
   label?: React.ReactNode;
   fieldHelp?: React.ReactNode;
 }): {
   labelId?: string;
-  validationIconId?: string;
+  validationId?: string;
   fieldHelpId?: string;
   ariaDescribedBy?: string;
```

Comment:
What do you think about doing this so we don't encounter any unexpected behaviour related to prop names?

```suggestion
  "aria-describedby"?: string;
```

Reply:
I think that it would lead to more confusion, as we would have to convert that prop to camelCase when we want to use or modify it.